girderformindlogger/api/v1/user.py

```diff
@@ -349,8 +349,8 @@ def setUserRelationship(self, id, rel, otherId, otherName):
 
     @access.public(scope=TokenScope.USER_INFO_READ)
     @autoDescribeRoute(
-        Description('Update a user\'s ID Code.')
-        .param('id', 'Profile ID', required=True)
+        Description('Add a new ID Code to a user.')
+        .param('id', 'Profile ID', required=True, paramType='path')
```

Comment:
@henryrossiter @shnizzedy  We can remove "required=True" option, because it is path param and it is required by default. But it's unnecessary though. Everything else looks good!